MAUI/Accordion/getting-started.md

```diff
@@ -328,13 +322,13 @@ private void SfAccordion_Expanded(object sender, Syncfusion.Maui.Accordion.Expan
 
 ### Collapsing Event
 
-The `Collapsing` event will be triggered when the expander control is being collapsed.It can cancel collapsing with `ExpandingAndCollapsingEventArgs` that contains the following property:
+The [Collapsing](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.Accordion.SfAccordion.html#Syncfusion_XForms_Accordion_SfAccordion_Collapsing) event will be triggered when the expander control is being collapsed.It can cancel collapsing with [ExpandingAndCollapsingEventArgs](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.Accordion.ExpandingAndCollapsingEventArgs.html) that contains the following property:

```

Comment:
Provided links are Xamarin APIs. Please update with MAUI link.

Reply:
Addressed the concern provided